students/synowiec_krzysztof/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,20 @@
+def main():
+    list = [int(x) for x in input().split()]
+    minIndex = 0
+    maxIndex = 0
+    for x in range(1, len(list)):
+        if list[x] > list[maxIndex]:
+            maxIndex = x
+            continue
+        elif list[x] < list[minIndex]:
+            minIndex = x
+
+    temp_max = list[maxIndex]
```

Comment:
There is common python trick to avoid temporary vars in such situations. 
You can do multiple assignment in single line:
` list[max_idx], list[min_idx] = list[min_idx], list[max_idx]`